merkle-tree/indexed/tests/tests.rs

```diff
@@ -595,40 +598,42 @@ pub fn functional_non_inclusion_test() {
 //         onchain_indexed_merkle_tree.root(),
 //         "environment setup failed relayer and onchain indexed Merkle tree roots are inconsistent"
 //     );
+//
+//     // All actors are using the initial state of indexing array.
 //     let actor_1_indexed_array_state = relayer_indexing_array.clone();
 //     let actor_2_indexed_array_state = relayer_indexing_array.clone();
-
+//     let actor_3_indexed_array_state = relayer_indexing_array.clone();
+//
 //     let (old_low_address_1, old_low_address_next_value_1) = actor_1_indexed_array_state
 //         .find_low_element(&address_1)
 //         .unwrap();
 //     let address_bundle_1 = relayer_indexing_array
 //         .new_element_with_low_element_index(old_low_address_1.index, &address_1)
 //         .unwrap();
 //     let change_log_index = onchain_indexed_merkle_tree.changelog_index();
-//     {
-//         let mut low_element_proof_1 = relayer_merkle_tree
-//             .get_proof_of_leaf(old_low_address_1.index, false)
-//             .unwrap();
-
-//         onchain_indexed_merkle_tree
-//             .update(
-//                 change_log_index.clone(),
-//                 address_bundle_1.new_element.clone(),
-//                 old_low_address_1.clone(),
-//                 old_low_address_next_value_1,
-//                 &mut low_element_proof_1,
-//             )
-//             .unwrap();
-//     }
-
-//     // getting parameters for the second actor with the pre update state
+//
+//     let mut low_element_proof_1 = relayer_merkle_tree
+//         .get_proof_of_leaf(old_low_address_1.index, false)
+//         .unwrap();
+//
+//     onchain_indexed_merkle_tree
+//         .update(
+//             change_log_index.clone(),
+//             address_bundle_1.new_element.clone(),
+//             old_low_address_1.clone(),
+//             old_low_address_next_value_1,
+//             &mut low_element_proof_1,
+//         )
+//         .unwrap();
```

Comment:
leftover?

Reply:
Not really. I know it's commented code, but we want to uncomment it once concurrency is fixed. I adjusted this code to my changes.

Reply:
Actually, I kicked it out now. I will try to fix the address MT concurrency in a separate PR.